src/geojson_types.jl

```diff
@@ -0,0 +1,350 @@
+abstract type GeoJSONT{D,T} end
+abstract type AbstractGeometry{D,T} <: GeoJSONT{D,T} end
+abstract type AbstractFeatureCollection{D,T} <: GeoJSONT{D,T} end
+
+"""
+    CRS(type::String, properties::Dict{String,Any})
+
+A Coordinate Reference System for compatibility.
+Should not be used, as it is not part of the GeoJSON specification.
+The CRS of a GeoJSON object is always WGS84.
+"""
+struct CRS
```

Comment:
Hmm... why does this need to exist?

Reply:
I believe there's a test example that has a crs. I tried to be somewhat backwards compatible in terms of the extra fields encountered in the old tests.

Reply:
Yeah it probably makes sense. Although we will need to explicitly not handle crs point order.